docs/4.3/enterprise/index.md

```diff
@@ -10,7 +10,6 @@ The table below gives a quick overview of the benefits of Teleport Enterprise.
 ---------|--------------
 |[Role Based Access Control (RBAC)](#rbac)|Allows Teleport administrators to define User Roles and restrict each role to specific actions. RBAC also allows administrators to partition cluster nodes into groups with different access permissions.
 |[Single Sign-On (SSO)](#sso)| Allows Teleport to integrate with existing enterprise identity systems. Examples include Active Directory, Github, Google Apps and numerous identity middleware solutions like Auth0, Okta, and so on. Teleport supports SAML and OAuth/OpenID Connect protocols to interact with them.
-|[Approval Plugins](workflow/index.md) | Plugins to approve of Deny escalated RBAC requests.
```

Comment:
Is this coming back in a different iteration?

Reply:
I'm not sure what happened with my git commit history, I cherry picked these into https://github.com/gravitational/teleport/pull/3858